src/test/resources/config/application-QueryStarterDefaults.yml

```diff
@@ -326,6 +326,7 @@ datawave:
           queryThreads: 16
           auditType: "NONE"
           logicDescription: "Retrieve graph edges matching the search term(s)"
+          maxConcurrentTasks: 10
```

Comment:
Not sure this is the default value we want across the board, but it gets junit to pass